docs/en/01-how-to-build/cross_build_aarch64.md

```diff
@@ -0,0 +1,108 @@
+# Ubuntu Cross Build aarch64
+
+mmdeploy chose ncnn as the inference backend for aarch64 embedded linux devices. There are two parts:
+
+Host
+
+- model conversion
+- cross build SDK and demo for embedded devices
+
+Device
+
+- Run converted model
+
+## 1. Model Convert on Host
+
+Refer to the doc to install [mmdeploy](../01-how-to-build/) and [mmcls](https://github.com/open-mmlab/mmclassification), and convert resnet18 for model package
+
+```bash
+$ export MODEL_CONFIG=/path/to/mmclassification/configs/resnet/resnet18_8xb32_in1k.py
```

Comment:
In this code block and other code blocks in the PR, `$` is inconvenient for users to copy, paste and run the commands.
It will show the following errors.
```
$: command not found
```

Reply:
done.